### DISCUSSION TITLE
Support for Personal & Enterprise Cloud EMU accounts

### DISCUSSION THREAD

**Discussion by ggiill** [community/community#8964] [Enterprise] [answered]
Hey!

My company just switched to Enterprise Cloud and many of my teammates and I would like to be able to add our EMU account to the mobile app in addition to our personal accounts.

It seems that the Enterprise Cloud EMU account occupies the personal account "slot".

Currently, it seems the mobile app only supports:
- A personal or Enterprise Cloud EMU account
- An Enterprise Server account

Ideally, it would support (at least):
- A personal account
- An Enterprise Cloud EMU account
- An Enterprise Server account

Thanks,
Gil

**Comment by brianlovin** [comment]
Thanks for the note @ggiill, agreed that it would be useful to connect both an EMU and personal account. I've flagged this to the team and added your note to a tracking issue, but this work isn't on our immediate roadmap.

**Reply by ex0b1t** [reply]
Hi @brianlovin any update on this feature?

**Comment by KevinMSampson** [comment]
I agree, this would be a great addition!

**Comment by ggiill** [comment]
Hey @brianlovin, just checking in: any movement on this one? Thanks!

**Reply by ggiill** [reply]
@candyho are you the right person to take this on? Looks like @brianlovin is no longer at GitHub. Thanks!

**Reply by candyho** [reply]
Yes! We are actively working on supporting multi-account on mobile, including EMU. Please stay tuned for more updates!

**Comment by eliperkins** [comment, marked as answer]
Hey @ggiill and folks, we've shipped functionality this week to allow you to log in with multiple accounts, like your personal and work accounts, at a the same time! [Read more about it in our Changelog post](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/). We built this functionality with EMU customers in mind. 🙇 

<img width="1200" alt="image" src="https://github.com/community/community/assets/1051453/d2ba902e-2199-4625-948a-532d64a6f44e">

Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.

**Reply by ggiill** [reply]
Thank you so much! 🙏

I was able to write this comment, upvote your update, and mark this issue as resolved on my 📱!
